SU-HSDO/stanford_mrc
# Mrcd8 172 visitor list

# READY FOR REVIEW

# Summary
- Visitor list view.
- moved field formatter into a field formatter plugin instead of a preprocess.

# Needed By (Date)
- End of sprint

# Urgency
- medium

# Steps to Test

1. Checkout branch
2. `drush updb -y`
3. verify the view on `/admin/structure/views/view/mrc_visitor/edit/all_visitors` looks like https://projects.invisionapp.com/share/DCEGPUZS4#/screens/265032968
4. verify the filters works correctly.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
